src/lev-report/charts/HourlyUsage.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { VictoryAxis, VictoryChart, VictoryLabel, VictoryLine, VictoryTheme } from 'victory';
+
+const hours = Array.from({ length: 25 }, (_, h) => h);
+
+const HourlyUsage = ({ traces }) => traces && traces.length &&
+<VictoryChart
+	padding={{ top: 10, bottom: 50, left: 50, right: 5 }}
+	domainPadding={20}
+	theme={VictoryTheme.material}
+	style={{ labels: { fontSize: 10 } }}
+	height={250}
+	width={300}
+>
+	<VictoryAxis
+		domain={[0, 24]}
+		tickValues={[0, ...hours.filter(n => !(n % 3))]}
+		tickFormat={t => t < 10 ? '0' + t : t}
+		style={{ tickLabels: { fontSize: 10 } }}
+	/>
+	<VictoryAxis dependentAxis />
+	{traces.map(t => <VictoryLine
+		key={t.name}
+		name={t.name}
+		colorScale="cool"
+		interpolation="natural"
+		data={t.data}
+		x="hour"
+		y="count"
+	/>)}
+</VictoryChart> || null;
+// basis, bundle, cardinal, catmullRom, linear, monotoneX, monotoneY, natural, step, stepAfter, stepBefore
```

Comment:
Should this comment be here? - I don't understand it.

Reply:
Ah yes, I've been meaning to give these a try - they're the other kinds of line path - I think "natural" interpolation works pretty well though... see the screenshot for details.